xbmc/pvr/dialogs/GUIDialogPVRTimerSettings.cpp

```diff
@@ -116,8 +116,9 @@ void CGUIDialogPVRTimerSettings::SetTimer(CFileItem *item)
   m_strTitle      = m_timerInfoTag->m_strTitle;
 
   const CDateTime now(CDateTime::GetCurrentDateTime());
-  m_startLocalTime    = m_timerInfoTag->IsStartAtAnyTime() ? now : InitializeDateTime(m_timerInfoTag->StartAsLocalTime());
-  m_endLocalTime      = m_timerInfoTag->IsEndAtAnyTime()   ? now : InitializeDateTime(m_timerInfoTag->EndAsLocalTime());
+  bool bFirstDay((m_timerType && m_timerType->SupportsFirstDay()) || (GetDateAsInt(m_timerInfoTag->FirstDayAsUTC()) != 0));
```

Comment:
OK - thanks for the warning - but I'm planning to un-do the 'first day'/'start/end' interactions. They are getting increasingly complex and I agree with you that they will probably break something later. (see my forum post http://forum.kodi.tv/showthread.php?tid=227026&pid=2057604#pid2057604)